gen_cert.py

```diff
@@ -1841,25 +1832,42 @@ def fontlist_with_style(a_style):
         yOffset = minYOffset + ((max_height - height) / 2)
         paragraph.drawOn(PAGE, GUTTER_WIDTH - (name_style.fontSize / 12), yOffset)
 
+
+        trans_success_text = ''
+        trans_interstitial = ''
+        trans_disclaimer = ''
+        trans_verify_text = ''
+
+        if self.locale in settings.DEFAULT_TRANSLATIONS:
+            trans_success_text = settings.DEFAULT_TRANSLATIONS[self.locale]['success_text']
+            trans_interstitial = settings.DEFAULT_TRANSLATIONS[self.locale]['grade_interstitial']
+            trans_disclaimer = settings.DEFAULT_TRANSLATIONS[self.locale]['disclaimer']
+            trans_verify_text = settings.DEFAULT_TRANSLATIONS[self.locale]['verify_text']
+
+        if self.course_translations and self.locale in self.course_translations:
+            trans_success_text = self.course_translations[self.locale].get('success_text', trans_success_text)
+            trans_interstitial = self.course_translations[self.locale].get('grade_interstitial', trans_interstitial)
+            trans_disclaimer = self.course_translations[self.locale].get('disclaimer', trans_disclaimer)
+            trans_verify_text = self.course_translations[self.locale].get('verify_text', trans_verify_text)
+
         # SECTION: Successfully completed
-        successfully_completed = u"has successfully completed a free online offering of"
         (fonttag, fontfile, completed_style) = font_for_string(
             fontlist_with_style(style_standard_text),
-            successfully_completed,
+            trans_success_text,
         )
 
         max_height = completed_style.leading
         max_width = MAX_GEN_WIDTH
         yOffset = 390     # distance from bottom of page (in points)
 
-        paragraph = Paragraph(successfully_completed, completed_style)
+        paragraph = Paragraph(trans_success_text, completed_style)
         width, height = paragraph.wrapOn(PAGE, max_width, max_height)
 
         paragraph.drawOn(PAGE, GUTTER_WIDTH, yOffset)
 
         # SECTION: Course Title
         course_name_string = self.long_course.decode('utf-8')
-        course_title = u"<b>{0}</b>".format(course_name_string)
+        course_title = u'<b>{0}</b>'.format(course_name_string)
```

Comment:
See note above about double-quotes for interpolated strings, as well as minimizing the diff here.